src/app/containers/StoryPromo/LinkContents.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useContext } from 'react';
+import moment from 'moment-timezone';
+import { shape } from 'prop-types';
+
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+
+import { ServiceContext } from '../../contexts/ServiceContext';
+import deepGet from '../../lib/json/deepGet';
+import formatDuration from '../../lib/utilities/formatDuration';
+import { storyItem } from '../../models/propTypes/storyItem';
+
+const LinkContents = ({ item }) => {
+  const {
+    translations: { media: mediaTranslations },
+  } = useContext(ServiceContext);
+
+  const isMedia = deepGet(['cpsType'], item) === 'MAP';
+  const headline = deepGet(['headlines', 'headline'], item);
+
+  if (!isMedia) {
+    return headline;
+  }
+
+  const type = deepGet(['media', 'format'], item);
+
+  // Always gets the first version. Smarter logic may be needed in the future.
+  const rawDuration = deepGet(['media', 'versions', 0, 'duration'], item);
+
+  // hilariously, this works. according to moment, null seconds == 0 seconds!
+  const duration = moment.duration(rawDuration, 'seconds');
+  const durationString = formatDuration(duration);
+  const isoDuration = duration.toISOString();
+
+  return (
+    // role="text" is required to correct a text splitting bug on iOS VoiceOver.
+    // eslint-disable-next-line jsx-a11y/aria-role
+    <span role="text">
+      <VisuallyHiddenText>{mediaTranslations[type]}, </VisuallyHiddenText>
+      <span>{headline}</span>
+      {rawDuration && (
+        <span>
+          <VisuallyHiddenText>
+            , <time dateTime={isoDuration}>{durationString}</time>
```

Comment:
The promo headline shouldn't use a `<time>` element. (Removes the need for a isoDuration)